Update 1294585866.rst (#125)
TYPO3-Documentation/TYPO3CMS-Exceptions: Documentation/Exceptions/1294585866.rst

Check your source code for occurrences of that class name. Usually happens if a
hook is registered with a wrong class name/reference.

> Solution 1: 

Reviewer: ```suggestion
Possible solution:
```

As there is currently only one solution. If there is a 1 I would also expect a 2.

> non Composer:

Reviewer: ```suggestion
Legacy installation:
```

> *   Try to use the TYPO3 backend Install Tool: Rebuild PHP Autoload Information

Reviewer: ```suggestion
*   Try to use the :guilabel:`Admin Tools > Maintenance` backend module: Rebuild PHP Autoload Information
```

> Composer:

Reviewer: ```suggestion
Composer-based installation:
```

Check your source code for occurrences of that class name. Usually happens if a
hook is registered with a wrong class name/reference.

Solution 1: 

> Try to use the TYPO3 backend Install Tool: Rebuild PHP Autoload Information

Reviewer 2: Only suitable/doable for non-composer installations. Composer installations needs run:

```shell
composer dump-autoload
```

or eventually `composer install (--no-dev)` to be safe.

Author: I have added now your proposal for composer.
